e2e/utils/test-utils.ts

```diff
@@ -32,10 +32,8 @@ export async function signInWithAccessToken(page: Page, tokenFilename = config.U
 
   // Once ready, initialize the token on the page (this is stored in local storage).
   // See sign-in.service.ts for details.
-  const navigationPromise = page.waitForNavigation({ waitUntil: ['load', 'networkidle0'] });
```

Comment:
`waitForNavigation` is not working well. Sometimes method fails. That's an issue in Puppeteer.